### ISSUE TITLE
Improve assertion failure output

### ISSUE THREAD

**Issue by mnahkies** [yaal-coop/scim2-tester#11]
At the moment I'm seeing the error `Server response payload validation error`, which unfortunately isn't very actionable. It would be better to output the pydantic validation errors.

```
ERROR check_service_provider_config_endpoint
  Server response payload validation error
  <Response [200 OK]>
```

**Comment by azmeuk** [comment]
Indeed, I have work to do on the displaying of the results. In the meantime, PRs are welcome.
Thank you for your report
